sampleModels/README.md

```diff
@@ -1,4 +1,4 @@
-Sample models are provided in as many of the following formats as possible:
+bSample models are provided in as many of the following formats as possible:
```

Comment:
Also, up to you, but it would not be a bad idea to replace each README.txt with README.md for consistency:

>  See the `README.txt` in each model's directory for usage restrictions.

Maybe even include the thumbnail and short description here also in the README.md for each model.